modules/flowable-app-rest/src/main/resources/static/docs/swagger-ui.js

```diff
@@ -21457,169 +21457,169 @@ function request(RequestConstructor, method, url) {
 module.exports = request;
 
 },{}],162:[function(require,module,exports){
-

```

Comment:
This are some new line changes which I can't seem to revert due to the setup of my repo. If someone can revert them I would have no issue